rpi-eeprom-config

```diff
@@ -160,7 +159,17 @@ def read_current_config():
     """
     Reads the configuration used by the current bootloader.
     """
-    return shell_cmd(['vcgencmd', 'bootloader_config'])
+    fw_base = "/sys/firmware/devicetree/base/"
+    nvmem_base = "/sys/bus/nvmem/devices/"
+    dev_path = ""
+
+    with open(fw_base + "/aliases/blconfig") as f:
```

Comment:
With the default rpi-update kernel I'm seeing an exception here because aliases/blconfig doesn't exist

Traceback (most recent call last):
  File "/usr/bin/rpi-eeprom-config", line 357, in <module>
    main()
  File "/usr/bin/rpi-eeprom-config", line 349, in main
    current_config, config_src = read_current_config()
  File "/usr/bin/rpi-eeprom-config", line 166, in read_current_config
    with open(fw_base + "/aliases/blconfig") as f:
IOError: [Errno 2] No such file or directory: '/sys/firmware/devicetree/base//aliases/blconfig'